tests/unit/config/jest.init.js

```diff
@@ -2,6 +2,8 @@ import { config } from '@vue/test-utils'
 import fetchMock from 'jest-fetch-mock'
 import 'regenerator-runtime/runtime'
 
+window.define = jest.fn()
```

Comment:
is this needed only for unit tests or should we have it inside the integration tests too?